6.5.100_degree.py

```diff
@@ -0,0 +1,73 @@
+import billboard
+import plotly.express as px
+import lyricsgenius
+import pandas as pd
+
+
+# Warning: This code will take a long time to run you can change the number of songs to 10 and not more than 100
+NUMBER_OF_SONGS = 100
+'''
+define a function that return the 100 songs from the Hot 100 of the Billboard using billboard.py
+'''
+def get_hot_100_songs():
+    chart = billboard.ChartData('hot-100')[0:NUMBER_OF_SONGS]
+    return chart
+
+
+'''
+define a function that chart from the Hot 100 of the Billboard as a parameter
+and return a list of the 100 songs lyrics using lyricsgenius
+'''
+def get_songs_lyrics(chart):
+    # Get the lyrics of each song from Genius
+    genius = lyricsgenius.Genius("x_ApCOemXCM80lez6bpRdYiuu--j3qsTjMOdl6AaEsKzCZe5o7zSS6qkdHyCerUT")
+    songs = []
+    for song in chart:
+        try:
+            lyrics = genius.search_song(song.title, song.artist).lyrics
+        except AttributeError:
+            lyrics = None
+        songs.append({'title': song.title, 'artist': song.artist, 'lyrics': lyrics})
+    return songs
+
+
+'''
+define a function that get a list of songs from the lyricsgenius as a parameter
+and count for each word in the song lyrics how many times it appears in the 100 songs
+the function  return a dictionary of the 100 most common words in the 100 songs and their count
+'''
+def count_words_in_lyrics(songs):
```

Comment:
Separate code to small functions- 
https://softwareengineering.stackexchange.com/questions/308108/when-is-it-appropriate-to-make-a-separate-function-when, -there-will-only-ever-be

The code in this function is not clean enough